common/src/main/java/org/geysermc/floodgate/config/FloodgateConfig.java

```diff
@@ -70,7 +70,16 @@ public boolean isProxy() {
         private boolean enabled;
         private boolean allowed;
         private long linkCodeTimeout;
+        private PlayerLinkDatabase database;
+    }
+
+    @Getter
+    public static class PlayerLinkDatabase {
         private String type;
+        private String database;
+        private String username;
+        private String password;
+        private String hostname;
```

Comment:
I feel like these should somehow be dynamically be included, maybe a separate config file just for the db as not all connections will have the same options.

Reply:
I was thinking the same.. but wasn't 100% sure how to handle it.. this was the fastest way to make it work cleanly